dot-net/hcx-integrator-sdk/Io.HcxProtocol/Docs/README.md

```diff
@@ -45,9 +50,21 @@ configObj.AuthBasePath = "http://a9dd63de91ee94d59847a1225da8b111-273954130.ap-s
 configObj.UserName = "<Email ID used for registration>";
 configObj.Password = "<Your Password>";
 configObj.EncryptionPrivateKey = "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEF=\n-----END PRIVATE KEY-----";
```

Comment:
Logtype is optional field or mandatory field. If mandatory, please add to the example configObj.